biggraphite/accessor.py

```diff
@@ -996,19 +1007,19 @@ def generate_values_aggregated(self):
                 if self.current_timestamp_ms != timestamp_ms:
                     # This needs to be optimized because in the common case
                     # there is absolutely nothing to aggregate.
-                    ts, point = self.run_aggregator()
+                    ts, point, agg_count = self.run_aggregator()
                     if ts is not None:
-                        yield (ts, point)
+                        yield (ts, point) if self.downsampled else (ts, point, agg_count)
```

Comment:
Make it clear in the docstring of the class that `count` will be returned if downsampled is set to False.